WORKSPACE

```diff
@@ -145,9 +145,14 @@ load("@rules_jvm_external//:defs.bzl", "maven_install")
 maven_install(
     artifacts = DAGGER_ARTIFACTS + get_maven_dependencies(),
     fetch_sources = True,
+    maven_install_json = "//:maven_install.json",
     repositories = DAGGER_REPOSITORIES + [
```

Comment:
When I tried setting `fail_if_repin_required` attribute here - 
```
maven_install(
    artifacts = DAGGER_ARTIFACTS + get_maven_dependencies(),
    fetch_sources = True,
    maven_install_json = "//:maven_install.json",
    fail_if_repin_required = True,
    repositories = DAGGER_REPOSITORIES + [
        "https://maven.fabric.io/public",
        "https://maven.google.com",
        "https://repo1.maven.org/maven2",
    ],
)
```

It threw an error - 
```
Starting local Bazel server and connecting to it...
ERROR: Traceback (most recent call last):
        File "/home/prayutsu/opensource/oppia-android/WORKSPACE", line 145, column 14, in <toplevel>
                maven_install(
        File "/home/prayutsu/.cache/bazel/_bazel_prayutsu/65c630ff6a7321024a3fd83ac1880b0b/external/rules_jvm_external/defs.bzl", line 20, column 5, in maven_install
                def maven_install(
Error: maven_install() got unexpected keyword argument: fail_if_repin_required
ERROR: error loading package '': Encountered error while reading extension file 'defs.bzl': no such package '@maven//': error loading package 'external': Could not load //external package

```

![Screenshot from 2021-06-07 18-38-33](https://user-images.githubusercontent.com/54636525/121022040-9d7fa000-c7bf-11eb-85d3-c263a55b34b9.png)

Reply:
Try a newer version of rules_jvm_external.

https://github.com/oppia/oppia-android/blob/f87e5047ef9da68fac4bc40168149ff50e1d4a09/third_party/versions.bzl#L112

Reply:
Got fixed, Thanks @fsharpasharp :) 
We were using quite an old version.